electron-app/src/views/settings/UserSecuritySettings.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <v-container id="settings-user-security">
```

Comment:
can we replace that with a class `user-security-settings`?

Reply:
Sure.